File: src/JavaScript/easy/string/problem2278.js
/**
 * @param {string} s
 * @param {character} letter
 * @return {number}
 */

// Not worked in Leetcode compiler

// var percentageLetter = function (s, letter) {
//     return ((s.match(new RegExp(letter, 'g') || []).length * 100) / s.length).toFixed(0)
// };

var percentageLetter = function (s, letter) {
    let countLetter = 0
    for (let i = 0; i < s.length; i++) {
        if ([...s][i] == letter) {
            countLetter++
        }
    }
    return Math.floor((countLetter * 100) / s.length)
};

File: src/Python/easy/string/problem2278.py
import math

class Solution:
    def percentageLetter(self, s: str, letter: str) -> int:
        countLetter = 0
        for i in range(len(s)):
            if s[i] == letter:
                countLetter += 1
        return math.floor((countLetter * 100) / len(s))


s = "foobar"; letter = "o"

solution = Solution()
solution.percentageLetter(s, letter)
